### PR TITLE
AddPowerSupport_python-memcached

### PR DESCRIPTION
**Pull request by santosh653**
Creating python-memcached.travis.yml with proposed changes.

### CODE REVIEW

File: travis-ymls/python-memcached.travis.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  - memcached
+before_install:
+  - |
+     if [[ "$TRAVIS_CPU_ARCH" = "ppc64le" ]]; then
+      sudo apt-get install memcached  
+     fi
+install: python setup.py install
+before_script: 
+  - python -m pip install -U pip
+  - python -m pip install flake8
```
Comment (gururajrkatti) [inline travis-ymls/python-memcached.travis.yml:39]: whats the reason "pip install -r test-requirements.txt" from original file removed ?

**Review by gururajrkatti — CHANGES_REQUESTED** [review]

**Comment by gururajrkatti** [comment]
@santosh653 any updates on this

**Comment by gururajrkatti** [comment]
@santosh653 any updates on this ?

**Comment by gururajrkatti** [comment]
@santosh653 can you pl. revert on this

**Comment by gururajrkatti** [comment]
@santosh653 any updates on this

**Comment by santosh653** [comment]
closing this PR.

**Pull request closed without merging**